docs/sources/features/datasources/jaeger.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Using Jaeger in Grafana"
+description = "Guide for using Jaeger in Grafana"
+keywords = ["grafana", "jaeger", "guide", "tracing"]
+type = "docs"
+aliases = ["/docs/grafana/latest/datasources/jeager"]
+[menu.docs]
+name = "Jaeger"
+parent = "datasources"
+weight = 2
++++
+
+# Using Jager in Grafana
+
+Grafana ships with built-in support for Jaeger, open source, end-to-end distributed tracing.
```

Comment:
```suggestion
Grafana ships with built-in support for Jaeger, which provides open source, end-to-end distributed tracing.
```

Reply:
`Jaeger, open source, end-to-end distributed tracing` is official short description for Jaeger so I sort of though it appropriate here.

Reply:
Well, their official description is not a sentence.